.mvn/maven.config

```diff
@@ -4,3 +4,6 @@
 --no-transfer-progress
 -Dmaven.test.error.ignore=true -Dmaven.test.failure.ignore=true
 -Dtycho.resolver.classic=false
+--fail-at-end
+-Dtycho-version=4.0.0-SNAPSHOT
+-s setup/tychoSnapshotSettings.xml -U
```

Comment:
please use one line for one command, Maven 3.9 will not support having everything in one line anymore.

Reply:
Done.